### DISCUSSION TITLE
vistual service and application in different namespace

### DISCUSSION THREAD

**Discussion by nikhilkhanpa** [argoproj/argo-rollouts#3946] [General]
we are trying to implement argo rollout /   flagger to update your deployment strategy to Canary / bluegreen. 

But, we could not find an option to specify namespace when we create rollout. 

In our case, application is in one namespace and virtual service is in another namespace. 

Do you have any suggestions over this issue ?

**Comment by kostis-codefresh** [comment]
For the built-in Istio support this doesn't seem to be possible at the moment. You should open an enhancement request

Alternatively you can use the Gateway API support of Istio and try with a HTTPRoute (that can exist in another namespace) https://rollouts-plugin-trafficrouter-gatewayapi.readthedocs.io/en/latest/quick-start/#step-5-create-a-rollout
